### PR TITLE
remove: Drash.Compilers.TemplateEngine

### LINKED ISSUE #430: [Code Removal] Drash.Compilers.TemplateEngine (Removal Date: 01-01-2021)
## Summary

See #399 and #424 for discussions surrounding deprecation and removal of this code.

_Note to team: November 6, 2020 is a tentative date. If we happen to finish the tasks later than November 6, 2020, that's fine._

## Tasks

- [x] November 6, 2020: Merge #428 (after_resource hook)
- [x] November 6, 2020: Merge #424 (deprecate Drash.Compilers.TemplateEngine)
- [x] November 6, 2020: Merge drashland/deno-drash-middleware#62 (Tengine - A template engine middleware)
- [x] November 6, 2020: Release deno-drash-middleware v0.6.0
- [x] November 6, 2020: Release deno-drash v1.3.0
- [x] January 1, 2021: Open PR to remove Drash.Compilers.TemplateEngine and all related tests
- [ ] January 1, 2021: Merge the above PR
- [ ] January 1, 2021: Release deno-drash minor version

### PR DESCRIPTION
**Pull request by crookse**
Closes #430 

## Summary

* Removes all instances of `template_engine` config
* Removes all instances of `views_path` config
* Removes all instances of `Drash.Compilers.TemplateEngine`
* Removes deprecation warning message related to template engine
* Removes tests related to the template engine
* Updates the React app (for Create App) to display as follows (to be just like the Vue app):
    Screen Shot 2020-12-26 at 04 49 32

### CODE REVIEW

File: src/http/response.ts [hunk 1]
```diff
@@ -91,47 +66,13 @@ export class Response {
...
-   *     if (!content) throw new Error(...)
-   *     this.response.body = content
-   *
-   * @returns The html content of the view, or false if the `views_path` is not
-   * set.
+   *   https://github.com/drashland/deno-drash-middleware/tree/master/tengine
    */
   public render(
     ...args: unknown[]
   ): Promise<boolean | string> | boolean | string {
```
Comment (ebebbington) [inline src/http/response.ts:75]: Should we not remove this method now?

File: tests/unit/http/response_test.ts [hunk 2]
```diff
@@ -34,52 +34,14 @@ Rhum.testPlan("http/response_test.ts", () => {
       },
     );
   });
+
   Rhum.testSuite("render()", () => {
-    Rhum.testCase("Returns false is `views_path` is falsy", () => {
+    Rhum.testCase("Returns false by default", () => {
```
Comment (ebebbington) [inline tests/unit/http/response_test.ts:39]: Noting this here due to my above comment

File: src/http/response.ts (hunk 1 above)
Comment (crookse) [reply src/http/response.ts:75]: we need this so that users can still use this.response.render()

Comment (ebebbington) [reply src/http/response.ts:75]: Don't suppose you could add that as a note for the method?

Comment (crookse) [reply src/http/response.ts:75]: ah right

Comment (crookse) [reply src/http/response.ts:76]: 78ae033d1b5b4bfae98f068ad0dbdf1a5c7c7073

**Review by ebebbington — APPROVED** [review]
LGTM

**Review by Guergeiro — APPROVED** [review]
Let's GO!

**Comment by crookse** [comment]
HAHA i'm slacking. i'll do it SOON.

**Review by saragee3 — APPROVED** [review]
chop CHOP ;)

**Pull request merged**